src/Html.php

```diff
@@ -1257,6 +1262,46 @@ public static function br(): Br
         return Br::tag();
     }
 
+    /**
+     * Generates video {@see Video} tag.
+     */
+    public static function video(): Video
+    {
+        return Video::tag();
+    }
+
+    /**
+     * Generates audio {@see Audio} tag.
+     */
+    public static function audio(): Audio
+    {
+        return Audio::tag();
+    }
+
+    /**
+     * Generates track {@see Track} tag.
+     */
+    public static function track(): Track
```

Comment:
Suggest add `src` as argument of this method. `src` is required attribute of `track` tag.